README.md

```diff
@@ -775,7 +776,7 @@ TODO: - does not use lib*v8 directly
 TODO: - does not parse data formats (files [images, video, audio,
 TODO:   xml, json, asn.1], network packets, structures, ...) from
 TODO:   an untrusted source.
-TODO: - does not open a port/socket
+TODO: - does not open a port/socket/dbus-endpoint or similar
```

Comment:
As discussed during the meeting: I always interpreted that sentence as "external endpoints" or "things that can be communicated to external. Otherwise, everything taking any input is a potential threat, so command line tool using any arguments (or taking input on stdin) would go under the same threat model? As said, similar to Unix socket (which is dbus afterall).

Also, on those, the client is the one to scan, even more than the service, so anything connecting as a client to dbus.

This is where I think the lines blur and I was thinking this part was really about "exposing something running on the machine to the outter world"?

Also, if we had those, we need the symetry IMHO on the template for filing the MIR.

Reply:
> I always interpreted that sentence as "external endpoints"

Yeah, as discussed in the call, that is a nice clarification. I like "external endpoint" and have rewritten it with that in mind.

>  Otherwise, everything taking any input is a potential threat

It is in some way, but that we already have in `does not parse data formats` and similar rules.

> Also, if we had those, we need the symmetry IMHO on the template for filing the MIR.

It is only a poor mans attack surface evaluation anyway to decide if it should go through a security review or not.
But yes, for symmetry we need matching statements (we have the ports) for the reporter.
That also helps them to explain if/why endpoints exist and skips us having to ask.

Reply:
I like this way more! Thanks for the change :)